### ISSUE TITLE
[Feature Request] Saving username in inapp browser login window

### ISSUE THREAD

**Issue by linshu** [nraboy/ng2-cordova-oauth#22]
It could be nice if the username is saved and not asked to reenter when the inapp browser login window is opened again even after the access token has expired.

This feature is like on the normal desktop browser, the username can be used for the next session even after the browser is closed or the access token is expired.

Thanks.

**Comment by derksacklowski** [comment]
+1

**Comment by stalniy** [comment]
In the last PR I've added ability to specify `browserOptions`, so you can overwrite defaults (check development branch):
```js
new CordovaOauth().logInVia(new Facebook(...), {
      clearsessioncache: 'no',
      clearcache: 'no'
})
```

**Comment by nraboy** [comment]
This has been merged to the `development` branch.  I'll see about getting it merged into `master` this weekend.

**Comment by nraboy** [comment]
Available as of 0.0.6.

Best,
